pages/linux/systemctl-list-units.md

```diff
@@ -0,0 +1,33 @@
+# systemctl list-units
+
+> List units that systemd currently has in memory.
+> See also: `systemctl list-unit-files` for listing installed unit template files.
```

Comment:
```suggestion
> See also: `systemctl list-unit-files` for listing installed unit files.
```
Are you sure it only applies to template files? Template files are the ones that end in an `@`